### DISCUSSION TITLE
Server Hello

### DISCUSSION THREAD

**Discussion by galdudi** [ben221199/WAPI#16] [Q&A]
Hi,
I would like to know, if connection working for you. I reach the stage of sending a "Client Hello", but get nothing back, no "Server Hello" no nothing else...

**Comment by ben221199** [comment]
Okay, where is your code?

**Reply by ben221199** [reply]
> I am talking about your code (Java). I just added edge_routing_info and client_static_keypair from verification process

Do you know if your phone numer is not banned?

**Reply by galdudi** [reply]
I am talking about your code (Java). I just added edge_routing_info and client_static_keypair from verification process

**Reply by galdudi** [reply]
Sorry about a new thread

**Reply by galdudi** [reply]
My phone is not banned, I still can work with it from bluestacks

**Reply by ben221199** [reply]
I mean, is it banned by WhatsApp during the registration?

**Reply by galdudi** [reply]
No, whatsapp sending me sms, and after that, ok record with edge_routing_info, that info with keypair from registration I'm using to make a connection, but after sending a client hello, I do not receive nothing from whatsapp server

**Comment by galdudi** [comment]


**Reply by ben221199** [reply]
Can you please reply under my post and not start a new thread?
